src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -149,7 +153,7 @@
                 return this.selectedReport && this.parameterValues.length
             },
             showChangelog: function () {
-                return !!this.selectedReport && this.metadata.changelog_types
+                return !!this.selectedReport && this.metadata?.changelog_types
```

Comment:
Do we need double negation here?

Reply:
No I suppose not

Reply:
Ah, yes we do, otherwise when both values are non-null we get an array instead of a boolean.